LocalSettings.php

```diff
@@ -6096,6 +6096,41 @@
 				APCOND_EMAILCONFIRMED,
 			),
 		),
+		'+igrovyesistemywiki' => array(
```

Comment:
this array is not closed

Reply:
@The-Voidwalker I don’t see what you are seeing

Reply:
The array needs to be closed on a newline added after line 6133